From 3ed33f899c5eb34bf1a1252e02d11bf4defcc79e Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Sat, 5 Nov 2022 20:43:30 +0100
Subject: [PATCH] Force recythonization

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e68fa7b..c08f66b 100644
--- a/setup.py
+++ b/setup.py
@@ -298,6 +298,6 @@ if __name__ == '__main__':
 
         for e in extensions:
             e.include_dirs.append(np.get_include())
-        metadata['ext_modules'] = cythonize(extensions, language_level=sys.version_info[0])
+        metadata['ext_modules'] = cythonize(extensions, language_level=sys.version_info[0], force=True)
 
     setup(**metadata)
-- 
2.37.1 (Apple Git-137.1)

