diff --git a/mdtraj/rmsd/include/util_arm.h b/mdtraj/rmsd/include/util_arm.h
index 5def97e6..10daba39 100644
--- a/mdtraj/rmsd/include/util_arm.h
+++ b/mdtraj/rmsd/include/util_arm.h
@@ -2,7 +2,7 @@


 static INLINE void aos_interleaved_store(float* p, float32x4_t x, float32x4_t y, float32x4_t z) {
-    vst3q_f32(p, float32x4x3_t{x, y, z});
+    vst3q_f32(p, (float32x4x3_t{x, y, z}));
 }

 static INLINE float32x4x3_t aos_deinterleaved_load(const float* S) {
