diff --git a/basesetup.py b/basesetup.py
--- a/basesetup.py	(revision a2ff6e23ae088dff5c16368eb32722548df4c213)
+++ b/basesetup.py	(date 1714337816106)
@@ -61,7 +61,7 @@
              self.compiler_args_sse2 = []
         self.compiler_args_sse3 = ['-mssse3'] if (self.sse3_enabled and not self.msvc) else []
         self.compiler_args_neon = []
-        self.compiler_args_warn = ['-Wno-unused-function', '-Wno-unreachable-code', '-Wno-sign-compare'] if not self.msvc else []
+        self.compiler_args_warn = ['-Wno-unused-function', '-Wno-unreachable-code', '-Wno-sign-compare', '-Wno-error=incompatible-function-pointer-types'] if not self.msvc else []

         if self.neon_enabled:
             self.compiler_args_sse2 = []
===================================================================
diff --git a/setup.py b/setup.py
--- a/setup.py	(revision a2ff6e23ae088dff5c16368eb32722548df4c213)
+++ b/setup.py	(date 1714338650720)
@@ -112,6 +112,7 @@
                     define_macros=[('USE_ZLIB', 1)],
                     library_dirs=zlib_library_dirs,
                     libraries=['z'],
+                    extra_compile_args=compiler_args
                     )

     dcd = Extension('mdtraj.formats.dcd',
